doc/hardware/at86rf212.rst

```diff
@@ -0,0 +1,105 @@
+.. at86rf212:
+
+AT86RF212 setup
+##################
+
+Category: radio
+Standard: IEEE 802.15.4
+Boards: MinnowBoard MAX, Galileo Gen 2
+
+Connection with MinnowBoard MAX board
+
+
+.. image:: _static/at86rf212-minnow.png
```

Comment:
@dbkinder This radio board does not include soldered pins, by soldering the board to Galileo PCB directly, we can get better electrical signal